crates/core/src/fnc/util/http/mod.rs

```diff
@@ -13,15 +13,16 @@ pub(crate) fn uri_is_valid(uri: &str) -> bool {
 
 fn encode_body(req: RequestBuilder, body: Value) -> RequestBuilder {
 	match body {
-		Value::Bytes(bytes) => req.header(CONTENT_TYPE, "application/octet-stream").body(bytes.0),
```

Comment:
Do we consider this removal a breaking change? We should be sure to mention the behaviour change in the documentation and the release note